plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/editors/elements/GraphicalFeatureModel.java

```diff
@@ -273,6 +273,17 @@ public void init() {
 		return Collections.unmodifiableList(features);
 	}
 
+	@Override
+	public List<IGraphicalFeature> getVisibleRelations() {
+		final List<IGraphicalFeature> features = new ArrayList<IGraphicalFeature>();
+		for (final IGraphicalFeature f : getFeatures()) {
+			if (!f.isCollapsed()) {
```

Comment:
Shouldn't this also check for hidden status? Like getVisibleFeatures() in the class FeatureModel does. If not, there should be a comment or the method should be renamed.